### PR TITLE
Feat/3914  document s3 storage

### PR DESCRIPTION
**Pull request by asf-algoo**
See #3914.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] If there are modifications in the Tracim configuration files (eg. `development.ini`), make sure it is documented in `backend/doc/setting.md` and notify relevant people of this change

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/doc/setting.md [hunk 1]
```diff
@@ -1,184 +1,40 @@
...
-
-You can set those parameters in INI configuration file (see `config_file_name`) or
-environnement variable (see `env_var_name`).
-
-The priority order is (from less to more priority):
-- default values
-- configuration file
-- environnement variables
-
-<!--- Maintainer: use tracimcli dev parameters list --template "| {env_var_name: <74}| {config_file_name: <63}| {config_name: <67}|"  with all apps enabled to update this list properly --->
```
Comment (inkhey) [inline backend/doc/setting.md:17]: You have removed the explaination to generate the list ?

**Review by raphj — APPROVED** [review]

File: backend/doc/setting.md (hunk 1 above)
Comment (raphj) [reply backend/doc/setting.md:17]: This is not the first time someone tries to remove this line. I already did once.

It looks like a leftover. Maybe we should just put it as regular text÷

Comment (PhilippeAccorsi) [reply backend/doc/setting.md:17]: I think also its good for all user to know what we need to do to generate this list.

Comment (grignards) [reply backend/doc/setting.md:17]: Sorry, I forgot to re-introduce it in `env_settings.md`, I'll do that.

**Review by inkhey — APPROVED** [review]

**Pull request merged**